classes/local/object_manipulator/checker.php

```diff
@@ -50,12 +50,16 @@ protected function get_query_name() {
     }
 
     protected function get_candidates_sql() {
-        $sql = 'SELECT f.contenthash
+        $sql = 'SELECT f.contenthash,
```

Comment:
purely for consistency I think the column order here should match the others ie id, hash then size

Reply:
updated